sqlglot/dialects/vertica.py

```diff
@@ -0,0 +1,40 @@
+from sqlglot import exp
+from sqlglot.dialects.dialect import Dialect
+from sqlglot.tokens import Tokenizer, TokenType
+from sqlglot.generator import Generator
+
+
+class Vertica(Dialect):
+    # Ignoring mypy errors for forward declarations
+    class Tokenizer(Tokenizer):  # type: ignore
+        QUOTES = ["'", '"']
+        IDENTIFIERS = ["`"]
+
+        KEYWORDS = {
+            **Tokenizer.KEYWORDS,
+            "INT64": TokenType.BIGINT,
+            "FLOAT64": TokenType.DOUBLE,
+            "VARCHAR": TokenType.STRING,
+            "DATE": TokenType.DATE,
+            "TIMESTAMP": TokenType.TIMESTAMP,
```

Comment:
```suggestion
```

Reply:
Could you please provide us why we need to remove Date and timestamp?

Reply:
They are redundant, you already have them in the base class